pkg/connector/ids.go

```diff
@@ -88,3 +90,26 @@ func (tc *TwitterClient) MakeEventSender(userID string) bridgev2.EventSender {
 		Sender:      MakeUserID(userID),
 	}
 }
+
+type MediaInfo struct {
+	UserID networkid.UserLoginID
+	URL    string
+}
+
+func MakeMediaID(userID networkid.UserLoginID, URL string) networkid.MediaID {
+	info := &MediaInfo{
+		UserID: userID,
+		URL:    URL,
+	}
+	id, err := json.Marshal(info)
```

Comment:
Media IDs should be as compact as possible. It could at least be version byte + uint64 user id + string (with length byte as prefix). If the URL has constant parts (which aren't at risk of changing), it could be parsed further too